New package: PRISMA v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: PRISMA
- Repository: https://github.com/cecoeco/PRISMA.jl
- Created by: @cecoeco
- Version: v0.0.1
- Commit: 2510281b7bff674a6d06423cd612d2399526b497
- Git reference: HEAD
- Description: Generate checklists and flow diagrams based on the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) statement.

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README.

As for the name, I don't necessarily have a problem with `PRISMA`. Generally speaking, I'm fine with pronounceable acronyms. If `Prisma` would be fine (as a "less systematic" package name), then so should `PRISMA`.

Although you might also go for a longer descriptive package name, and we might be able to give you some recommendations if you add some more documentation to the README.  In any case, `PRISMA` would need a manual merge by a registry maintainer.

cecoeco commented:
Thank you for reviewing my package, goerz. I was able to add a longer description and some documentation in the readme like you said. A little background: the package name comes from the Preferred Reporting Items for Systematic reviews and Meta-Analyses (PRISMA) statement.

goerz commented:
[noblock] Thanks for updating the README. That looks good enough to me for registration.

Like I said, I'm fine with the name. IMO, acronyms that are deliberately designed to result in a pronounceable name or word are fine, and fall in the allowed category of "less systematic names", independent of their capitalization.

That opinion might not be universal among the registry maintainers, so you may still get some pushback. But if they don't manage to convince you, it'll probably get merged eventually if you stick to your guns. But in any case, you'll have to ask for a manual merge on Slack after the 3-day waiting period ends.

You also still have to fix the `compat` issues, no matter what. When you do, just retrigger the registration. Do not change the `version` of the package. As long as the version, UUID, and package name remain the same, retriggering the registration should update this existing PR instead of creating a new one.

carstenbauer commented:
[noblock]

I'm also fine with the name. You still need to fix the compat entries though.

JuliaHubRegistrator committed:
New package: PRISMA v0.0.1

UUID: 7b67a8d2-c4f5-4933-b91c-0fc427024db5
Repo: https://github.com/cecoeco/PRISMA.jl.git
Tree: cd68b3460d69b17a10bc8462ca8b6f69849239dd

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

cecoeco commented:
Dependencies now have compat entries.
